SPECS/expat/expat.spec

```diff
@@ -48,24 +51,31 @@ make %{?_smp_mflags} check
 
 %post	-p /sbin/ldconfig
 %postun	-p /sbin/ldconfig
+
 %clean
 rm -rf %{buildroot}/*
+
 %files
 %defattr(-,root,root)
 %license COPYING
 %doc AUTHORS Changes
 %{_bindir}/*
-%{_mandir}/man1/*
 
 %files devel
 %{_includedir}/*
 %{_libdir}/pkgconfig/*
 %{_libdir}/libexpat.so
+%{_libdir}/cmake/expat-%{version}
 
 %files libs
 %{_libdir}/libexpat.so.*
 
 %changelog
+* Fri Nov 19 2021 Max Brodeur-Urbas <maxbr@microsoft.com> - 2.4.1-1
+- Update to 2.4.1
+- License verified
+- Removed manfile generation to avoid circular dependency.
```

Comment:
Is this change included? I could not find it.

Reply:
I should be more specific here, I removed the reference to man files as generating them in 2.4.1 causes a circular dependency. Will update the changlog to be more clear.